android/build.gradle

```diff
@@ -14,8 +14,14 @@ android {
   }
 }
 
+allprojects {
+  repositories {
+    jcenter()
+    maven { url "$projectDir/../../react-native/android" }
```

Comment:
like this : https://github.com/react-native-community/react-native-svg/blob/master/android/build.gradle